src/main/ccl/cclutmock_table.inc

```diff
@@ -706,10 +699,11 @@ subroutine public::cclutConstructDBSetStatement(cclutTableIndex, cclutColumnInde
     declare cclutMockColumnName = vc with protect, noconstant("")
 
     if (cclutColumnIndex > CCLUT_COLUMN_COUNT)
+        call parser(" reset")
+        call parser(" reset")
```

Comment:
wish we had a mutation testing tool 
(is there a test proving the first reset is necessary?)

Reply:
There is not a test that explicitly checks that parser was called with the words "reset", but it is indirectly tested through the TOO_MANY_COLUMNS test (the test that goes down this conditional path) because if it is excluded, then the next parser statement (execute null go) will generate an error which would cause the test to fail.